### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ShakedZrihen**

### CODE REVIEW

**Review by slowHand14 — COMMENTED** [review]
nb,

**Review by slowHand14 — COMMENTED** [review]
m,mnm

**Review by yeelali14 — COMMENTED** [review]
test

File: README.md [hunk 1]
```diff
@@ -17,3 +17,6 @@ This project was generated with [Angular CLI](https://github.com/angular/angular
 To get more help on the Angular CLI use `ng help` or go check out the [Angular CLI Overview and Command Reference](https://angular.io/cli) page.
 
 #PR
+
+
+LOLIPOP
```
Comment (nogaZohar) [inline README.md:22]: please use lower case here

**Review by nogaZohar — CHANGES_REQUESTED** [review]
see comment

File: README.md (hunk 1 above)
Comment (yeelali14) [reply README.md:22]: test

Comment (yeelali14) [reply README.md:22]: works?

**Pull request closed without merging**